sklearn/tree/tests/test_tree.py

```diff
@@ -2631,12 +2649,11 @@ def test_regression_tree_missing_values_toy():
     https://github.com/scikit-learn/scikit-learn/issues/28254
     """
 
-    # With this dataset, the missing values will always be sent to the left child
-    # at the first split. The leaf will be pure.
-    X = np.array([np.nan, np.nan, 3, 4, 5, 6]).reshape(-1, 1)
+    # With certain datasets, the missing values will always be sent to the left,
```

Comment:
you can remove the comment actually since this related the `X` so it looks a bit weird here now.

Reply:
Removed.